packages/server/src/websocket/websocket.service.ts

```diff
@@ -176,7 +182,8 @@ export class WebsocketService {
 
         if (token && messageId) {
           this.logger.debug(`[addMessage] Push notification parameters token: ${token}; MessageId: ${messageId}`)
-          await this.sendPushNotification(token, messageId)
+          //await this.sendPushNotification(token, messageId)
+          await this.fcmSendNotificationService({ token, messageId })
```

Comment:
```suggestion
          await this.fcmSendPushNotification({ token, messageId })
```

So in the end we are using only FCM. This makes sense in current implementation, because the token we receive from the mediator at the moment is an FCM token.

I think that if we want to support both (or an arbitrary number of notification methods), the AddMessage interface should be modified in such a way that, instead of receiving a string for the token, it receives an object that specifies the type. For instance:

pushNotificationToken: { type: string, value: string }
Where valid types are (at the moment): 'apn', 'fcm'.

Reply:
Solved!